client/src/pages/Notes.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import { Button, Icon } from 'semantic-ui-react';
+import 'draft-js/dist/Draft.css';
+
+import { getNotes } from '../utils/apiWrapper';
+import { notesColumnDefs } from '../utils/tableHelpers';
+import Loading from '../components/ui/Loading';
+import Page from '../components/layout/Page';
+import Table from '../components/table/Table';
+
+function Notes() {
+  const [notes, setNotes] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const history = useHistory();
+
+  useEffect(() => {
+    (async () => {
+      const {
+        data: { result },
+      } = await getNotes();
+
+      setNotes(result ?? []);
+
+      setIsLoading(false);
+    })();
```

Comment:
Im assuming this calls the async function immediately, for readability purposes though, it might be helpful to name the function and call it separately.